services/frontend-service/src/ui/components/SideBar/SideBar.tsx

```diff
@@ -200,6 +202,13 @@ type SideBarListItemProps = {
 export const SideBarListItem: React.FC<{ children: BatchAction }> = ({ children: action }: SideBarListItemProps) => {
     const { environmentLocks, appLocks } = useAllLocks();
     const actionDetails = getActionDetails(action, appLocks, environmentLocks);
+    var displayVersion = '';
+    try {
+        const release = useReleaseOrThrow(actionDetails.application ?? '', actionDetails.version ?? 0);
+        displayVersion = release.displayVersion;
+    } catch (error) {
+        // continue without displayVersion
+    }
```

Comment:
```suggestion
    var displayVersion = useRelease(actionDetails.application ?? '', actionDetails.version ?? 0).displayVersion | '';
```
Would something like this be possible?
Why do we have useRelease throwing errors?

Reply:
I agree with João. Since we do not want to throw an error, we can simply call useRelease

Reply:
Unfortunately, useRelease is not exported but I can create a helper function to move this logic

Reply:
```suggestion
    const displayVersion => {
        try {
            const release = useReleaseOrThrow(actionDetails.application ?? '', actionDetails.version ?? 0);
            return release.displayVersion;
        } catch (error) {
            return ''
        }
    }
```
I'd go with something like this. Not entirely sure the syntax is right or if there's some typescript error that might arise from this but I think it's cleaner and keeps the best of both worlds (const and error handling)